src/app/products/manage-delinquency-buckets/delinquency-bucket/edit-bucket/edit-bucket.component.ts

```
@@ -114,12 +124,45 @@ export class EditBucketComponent implements OnInit {
-      name: [
-        { value: this.delinquencyBucketData.name, disabled: true },
-        Validators.required
-      ]
-    });
+    if (this.isRegularBucket) {
+      this.bucketForm = this.formBuilder.group({
+        name: [
+          { value: this.delinquencyBucketData.name, disabled: true },
+          ,
+          Validators.required
+        ]
```

Biome (2.4.7)

[error] This array contains an empty slots..

(lint/suspicious/noSparseArray)